repair/row_level.cc

```diff
@@ -2948,7 +2949,8 @@ class row_level_repair {
                 if (table_dropped) {
                     throw replica::no_such_column_family(_shard_task.get_keyspace(),  _cf_name);
                 } else {
-                    throw std::runtime_error(format("Failed to repair for keyspace={}, cf={}, range={}", _shard_task.get_keyspace(), _cf_name, _range));
+                    throw nested_exception(std::make_exception_ptr(std::runtime_error(format("Failed to repair for keyspace={}, cf={}, range={}", _shard_task.get_keyspace(),
```

Comment:
I see two people have already approved this patch, so I will merge it, but something bothered me here: Unfortunately, we have three "versions" of nested_exception: There is std::nested_exception, there is seastar::nested_exception, and there is Scylla's util::internal::nested_exception exposed by the make_nested_exception_ptr(). This state of affairs is super confusing. I don't understand why we need it all these versions, but I think we should at least make the version we are using more explicit. In this case I believe you're using `seastar::nested_exception` so I would have used this qualified name.

Reply:
I see that @denesb already beat me to merging this PR while I was writing my comment ;-)

Reply:
I opened https://github.com/scylladb/scylladb/issues/15801 on the nested_exception duplication (regardless of the fact that I think we should have written here `seastar::nested_exception` to at least make clear which of the three variants we are using here).